src/legacy/ui/public/saved_objects/components/saved_object_finder.tsx

```diff
@@ -142,8 +169,10 @@ class SavedObjectFinder extends React.Component<SavedObjectFinderProps, SavedObj
       items: [],
       isFetchingItems: false,
       page: 0,
-      perPage: props.initialPageSize || props.fixedPageSize || 15,
+      perPage: props.initialPageSize || props.fixedPageSize || 10,
```

Comment:
Can you make sure that the page size selector doesn't show if the list doesn't have that many items?

<img width="297" alt="screen shot 2019-03-07 at 14 41 54 pm" src="https://user-images.githubusercontent.com/549577/53984426-4160d980-40e7-11e9-82f2-c472566e3ea5.png">